### ISSUE TITLE
[bug] New contributors table is showing malformatted data

### ISSUE THREAD

**Issue by jberkus** [cncf/devstats#43]
[See here](https://etcd.devstats.cncf.io/d/52/new-contributors-table?orgId=1)

Not sure what's supposed to be in those columns, but I am sure that's not supposed to be it.

**Comment by lukaszgryglicki** [comment]
Fixed, thanks for reporting @jberkus

**Comment by lukaszgryglicki** [comment]
I'm now checking other projects and will eventually apply a similar fix on them.

**Comment by lukaszgryglicki** [comment]
FYI - this happens in all projects, see my update related to this:
```
Wip: fix permanently in all projects: [#43 (DevStats).](https://github.com/cncf/devstats/issues/43)
Done: added util scripts to mass update all other projects dashboards sourcing one updated in the UI + docs for this (allows replacing on hits/non-hits for particular signature + updating both project key names and full names).
Done: added a script that generates bash associative array of all projects key-name mapping that can be later included in other shell scripts that needs to know both project key and its name (like the one above).
Wip: regenerating all grafanas on prod to include fix (migration) to table panel for new contributors dashboard - all projects (170+).
```

And changes from those commits: 
- https://github.com/cncf/devstats/commit/7b1c79c311b44d2a2f40760e753aeebe9d449a64
- https://github.com/cncf/devstats/commit/481b2bee7f53c2c2df2ee108eb94aecfd2d29670
- https://github.com/cncf/devstats/commit/d92d6140f9309b72e76d2f202945faf69a49d74d

Will update once all is finished.

**Comment by lukaszgryglicki** [comment]
All dashboards fixed, thanks for reporting @jberkus
